user_modules/shared_testcases/commonSteps.lua

```diff
@@ -527,5 +529,43 @@ function commonSteps:DataBaseQuery(DBQueryV)
     return false
 end
 
+-- 19.ActivateAppInSpecificLevel
+--! @brief Activate application with specific hmi level
+--! @param HMIAppID - app ID
+--! @param hmi_level - level for activation
+function commonSteps:ActivateAppInSpecificLevel(self, HMIAppID, hmi_level)
```

Comment:
@VVeremjova Currently this function can be used to set application to a 'FULL' level
And it's no possible to other ones ('LIMITED', 'BACKGROUND', 'NONE')